node/typings/v1typings.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module 'v1typings' {
```

Comment:
why not declaring the route specific typings in the user route datasource file ?